eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -340,6 +340,42 @@
         "code": "java.method.added",
         "new": "method com.azure.core.http.policy.HttpLogOptions com.azure.core.http.policy.HttpLogOptions::setResponseLogger(com.azure.core.http.policy.HttpResponseLogger)",
         "justification": "New method included in azure-core."
+      },
+      {
+        "code": "java.annotation.attributeValueChanged",
+        "old": "method byte[] com.azure.security.keyvault.keys.models.JsonWebKey::getDq()",
+        "new": "method byte[] com.azure.security.keyvault.keys.models.JsonWebKey::getDq()",
+        "justification": "Annotation moved to different package was package-private.",
```

Comment:
Re-word this to clarify that this isn't a breaking change - "Package-private Base64UrlJsonDeserializer class moved from models package to implementation package Base64UrlJsonDeserializer"